conf/sample.ini

```diff
@@ -397,6 +397,7 @@
 ;header_property = username
 ;auto_sign_up = true
 ;ldap_sync_ttl = 60
+;sync_ttl = 60
```

Comment:
You can remove ldap_sync_ttl from this file as we do not want new users to use it.

Maybe remove it from docs as well